static/js/common.js

```diff
@@ -58,8 +58,25 @@ function password_quality(password, bar, password_field) {
     return acceptable;
 }
 
+function password_warning(password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
```

Comment:
I am not sure why we are introducing redundancy. But interestingly the value of `PASSWORD_MIN_ZXCVBN_QUALITY` is 0.5 in settings and 0 in the password_quality function in common.js.